### ISSUE TITLE
Findings for Accessibility Staging Review for Medical Device Re-ordering Tool

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#9859]
**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

This epic will contain the accessibility staging review findings issue tickets.

## Point of Contact

**VFS Point of Contact:** Jennifer

cc @1Copenut — Ready for your review. 

> NOTE: 
> There are a number of bugs that prevent me from making it through the entire form logged in as all of the users. When logged in as 228, where there are no items available for re-order, I am able to get to the Review screen, but nothing is there. Logged in as 14, I am unable to proceed past the screen noted in [comment 8](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9859#issuecomment-640746257).

> There are a number of bugs and design issues noted here in the comments, rather than ticketed. I'm not sure they are the purview of accessibility, or if they are design or else.

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions.

**Comment by jenstrickland** [comment]
This is a running tally of unusual items noticed during the audit.

1. User 228 - no last 4 of SSN; no Date of birth; no Gender

**Comment by jenstrickland** [comment]
2. Design issue: On the introduction page, the bottom 'alert box' that encourages users to "Sign in to your account" has a link instead of the usual button. This **should** be a button, for consistency.

  ![Screen Shot 2020-06-08 at 10 20 04 AM](https://user-images.githubusercontent.com/57469/84041676-1ad8c080-a972-11ea-8656-d105ab53e34e.png)

**Comment by jenstrickland** [comment]
3. For the user who has an email address on file, the "Re-enter email address" field should not be pre-populated. A user should have to enter that to confirm the email address is correct. At present, the confirmation field is populates from the system.

**Comment by jenstrickland** [comment]
4. [Bug]: Something isn't being populated on sign-in, as shown in screen shot.


![Screen Shot 2020-06-05 at 5 00 18 PM](https://user-images.githubusercontent.com/57469/84055489-e7eaf880-a982-11ea-88d3-ee1895004efc.png)

**Comment by jenstrickland** [comment]
5. [Design/Bug]: The repetition of shipping address and email address should be addressed.

![Screen Shot 2020-06-08 at 12 30 21 PM](https://user-images.githubusercontent.com/57469/84056327-13221780-a984-11ea-91b1-0cea85b70a67.png)

**Comment by jenstrickland** [comment]
6. [Bug/Design] - Last week I was told there is a bug that is causing this user scenario to proceed to the review screen. 

![Screen Shot 2020-06-08 at 12 34 24 PM](https://user-images.githubusercontent.com/57469/84056611-72802780-a984-11ea-90d5-ef7dbd66156f.png)

**Comment by jenstrickland** [comment]
7. [Bug] When I submit the application, the screen renders with only the header and footer. The entire content area doesn't appear.

**Comment by jenstrickland** [comment]
8. [Bug] I am unable to continue past this screen, logged in as user 14. 

![Screen Shot 2020-06-08 at 12 48 42 PM](https://user-images.githubusercontent.com/57469/84057927-72812700-a986-11ea-9fb8-cc15562e347f.png)

**Comment by bradl3yC** [comment]
Bug 1 - this is a data issue - the mock user does not have this data
Bug 3 - we cannot get this confirmation field to pre-populate - This may have been the autocomplete on your browser

**Comment by bradl3yC** [comment]
@jenstrickland @leahkeeler can we close this since the tool is released?

**Comment by jenstrickland** [comment]
We don't usually close it until all the issue tickets are remediated.

**Comment by leahkeeler** [comment]
HI @jenstrickland - Can we close this ticket?

**Comment by jenstrickland** [comment]
Hi @leahkeeler - It's up to you. There is still one BAM2 ticket open, and the rest are platform's responsibility.

**Comment by jilladams** [comment]
All tix in the epic are closed. Closing.
